src/components/RemediationTable.scss

```diff
@@ -26,3 +26,7 @@
 .ins-c-table__toolbar .ins-c-skeleton {
     margin-left: auto;
 }
+
+.isNotEntitled {
```

Comment:
just for consistency, can we update this to `ins-m-not-entitled`?

Reply:
Yep!